### PR TITLE
Example keyword for scenario

### PR DESCRIPTION
**Pull request by aslakhellesoy**
## Summary

I'm proposing we start calling scenarios examples. This PR introduces a new `Example` keyword to replace the `Scenario` keyword. The `Scenario` keyword is still supported for backwards compatibility.

## Details

I have done my best to add translations of `Example` to the existing 60-70 or so languages, using Google translate. I'm sure I've made some mistakes, and I hope native speakers will point this out. 

Languages that don't distinguish between singular and plural of `Example` have not been changed because this would introduce an ambiguity with the existing `Examples` keyword (which is used underneath `Scenario Outline`). `Scenario Outline` itself has been translated to `Example Outline`.

## Motivation and Context

The BDD community is increasingly using the term "example" to describe *examples of behaviour*. This trend has been accelerated by the adoption of the [Example Mapping](https://cucumber.io/blog/2015/12/08/example-mapping-introduction) technique.

I hope that this change will encourage BDD practitioners to use a more consistent terminology. I realise existing literature will point people towards `Scenario` for several years, but at least this makes it possible for people to use `Example`. I know I will.

### CODE REVIEW

**Comment by tooky** [comment]
👍 great Aslak!

Can the parser use context to differentiate between the different use of Example in those languages without a plural form?

**Comment by aslakhellesoy** [comment]
> Can the parser use context to differentiate between the different use of Example in those languages without a plural form?

Hm - I think it might actually, with some changes to the parser. I suggest we don't worry about it for now and revisit it when/if people's craving for `Example` in their own language grows stronger.

**Comment by charlierudolph** [comment]
For the “Example Outline”, is there then going to be “Examples” under it? Might be better to have a different keyword for that. Maybe “Example Data” or “Data” or something else?

**Comment by mattwynne** [comment]
I don't love "Example Outline". I suggest we take that out of scope of this change and focus on just adding "Example" as a synonym of "Scenario".

**Comment by aslakhellesoy** [comment]
I don't love "Example Outline" either, but is it any worse than "Scenario Outline"? Both synonyms exist, and I think it's important to offer consistency to those who prefer it.

I'll see if we can just remove the special keyword altogether, and allow "Examples" under "Scenario". The context should be sufficient to decide whether to interpret it as a scenario or scenario outline. This would happen in a separate PR.

**Comment by mlvandijk** [comment]
Dutch translation is fine! Will try to find some native speakers in our very international IT departments to check some other languages ;)

**Comment by aslakhellesoy** [comment]
@mattwynne with https://github.com/cucumber/cucumber/pull/353 there is no more `Scenario Outline` or `Example Outline`, just `Example`.

**Comment by mlvandijk** [comment]
Macedonian approved by coworker ;)

**Comment by mlvandijk** [comment]
Chinese also approved by coworker ;)

**Comment by mlvandijk** [comment]
Italian improvement with help of Italian coworker ;) Otherwise approved.

**Comment by aslakhellesoy** [comment]
Grazie mille!

**Comment by gasparnagy** [comment]
After thinking about a lot, I actually have to say that I like `Scenario` and would suggest not to replace it with `Example`. Over the years the term "Scenario" has become the notation of the core requirement elements used by BDD. Essentially scenario describes an example, but it does more: by expressing a sentence by the steps using the G/W/T/And/But keywords, actually it tells a **story**... a scenario of the behaviour of the application. It is a well identifiable BDD term and by replacing it with such a very generic word, like example we cause more confusion.

For me example is that rough thing that I write onto an index card during an example mapping session. And when I later phrase it with G/W/T, it becomes a scenario, that describes the behavior of a particular rule. That is clear to me and have not seen much confusion about this in my courses...

File: gherkin/CHANGELOG.md [hunk 1]
```diff
@@ -10,10 +10,32 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 ## [Unreleased] - In Git
 
 ### Added
+<<<<<<< HEAD
```
Comment (xtrasimplicity) [inline gherkin/CHANGELOG.md]: Hey @aslakhellesoy, are you aware of the conflict here? :)

Comment (aslakhellesoy) [reply gherkin/CHANGELOG.md]: Well spotted! Will fix.

Comment (xtrasimplicity) [reply gherkin/CHANGELOG.md]: There's also another further down, in the Perl changelog.

File: gherkin/perl/CHANGES [hunk 2]
```diff
@@ -10,10 +10,32 @@ This document is formatted according to the principles of [Keep A CHANGELOG](htt
 ## [Unreleased] - In Git
 
 ### Added
+<<<<<<< HEAD
```
Comment (xtrasimplicity) [inline gherkin/perl/CHANGES]: Not sure if you've seen this, but there's some here, too, @aslakhellesoy.

**Pull request merged**